--- "kolibri/utils/server.py" 2021-11-19 11:23:25.360732982 +0100
+++ "kolibri/utils/server.py" 2021-11-19 11:23:29.248766236 +0100
@@ -71,7 +71,7 @@
 DAEMON_LOG = os.path.join(conf.LOG_ROOT, "daemon.txt")
 
 # Currently non-configurable until we know how to properly handle this
+LISTEN_ADDRESS = "127.0.0.1"
-LISTEN_ADDRESS = "0.0.0.0"
 
 status_messages = {
     STATUS_RUNNING: "OK, running",
@@ -646,8 +646,8 @@
         service_plugin.subscribe()
 
         # Setup zeroconf plugin
+        # zeroconf_plugin = ZeroConfPlugin(self, port)
+        # zeroconf_plugin.subscribe()
-        zeroconf_plugin = ZeroConfPlugin(self, port)
-        zeroconf_plugin.subscribe()
 
         signal_handler = SignalHandler(self)
 
